pkg/network/image-pull-policy.go

```diff
@@ -0,0 +1,54 @@
+package network
+
+import (
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+const defaultImagePullPolicy = "IfNotPresent"
```

Comment:
should you use `v1.PullIfNotPresent` instead of a literal here?

Reply:
Fixed